packages-content-model/roosterjs-content-model-editor/lib/modelApi/table/applyTableFormat.ts

```diff
@@ -43,15 +48,16 @@ export function applyTableFormat(
         };
 
         const bgColorOverrides = updateBgColorOverrides(rows, !keepCellShade);
+        const vAlignOverrides = updateVAlignOverrides(rows);
```

Comment:
These two functions can be merged into one, and no need to parse metadata for each cell twice

Reply:
Then let it directly return metaOverrides